alica_engine/include/engine/planselector/TaskAssignment.h

```diff
@@ -31,31 +31,30 @@ class PartialAssignmentPool;
  * Represents an instance of an assignment problem for one plan or a plantype.
  * All parameters, which are static for this problem, are stored here.
  */
-class TaskAssignment : virtual public ITaskAssignment {
+class TaskAssignment final : public ITaskAssignment {
```

Comment:
do we even need an ITaskAssignment class to inherit from?

Reply:
Currently it bridges between TaskAssignment and PartialAssignment. We may be able to lift that though.